CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
-## Changelog
+### v3.0.0-alpha0 (18-May-2017)
+
+Storybook 3.0 is our first fully community-driven release! Notable changes:
+- Move from `@kadira` to `@storybooks` org across [github](https://github.com/storybooks/storybook/), [npm](FIXME), [docs](https://storybooks.js.org/)
+- Upgrade to Webpack2 [PR637](https://github.com/storybooks/storybook/pull/637)
```

Comment:
This is a bit nitpicky, and really doesn't matter, but you can just include the URL and github will shorten it to a nicer looking link.

Example:
https://github.com/storybooks/storybook/pull/637
https://github.com/storybooks/storybook-addon-graphql/pull/19

```
https://github.com/storybooks/storybook/pull/637
https://github.com/storybooks/storybook-addon-graphql/pull/19
```

I think it's a slightly nicer format when viewing issues on github, and it falls back to the correct link when viewing code outside of github.

I'm totally fine using the `[PRXXX](link)` format if that's the desired format, but since it looks kind of inconsistent in the changelog's history, I thought I'd bring it up.